### DISCUSSION TITLE
GUIDE TO FIX WINGETUI NOT LAUNCHING

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2073] [Announcements]
# WingetUI 3.0.2 has been released, fixing most of the crashes. You may download it from [here](https://github.com/marticliment/WingetUI/releases/download/3.0.2/WingetUI.Installer.exe)

WingetUI 3.0.2 should not crash anymore. Apologies for the inconveniences and for the incompatibilities with .NET 8.0.3 and 8.0.4

------------------------------------------------------

## If WingetUI is not launching, do the following steps
1. Go to *Windows Settings* -> *Apps* -> *Installed Apps*
2. Uninstall the following packages (some of them may not be present on your machine):
   a) `WingetUI`
   b) `Windows App Runtime Main`
   c) `Windows App Runtime DDLM 5xxx.xx.xxxx.x`
   d) `Microsoft Windows App Runtime Singleton`
3. Download and install WingetUI from here: https://github.com/marticliment/WingetUI/releases/download/3.0.1/WingetUI.Installer.exe
4. Quit WingetUI and return to Windows Settings
5. Uninstall the following packages:
   e) `Microsoft .NET Runtime 8.0.X (x64)`
   f) `Microsoft .NET SDK 8.0.X (x64)`
6. Install .NET Desktop Runtime 8.0.2 using the following URL: https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-8.0.2-windows-x64-installer

## If the steps above do not work, please downgrade to version 2.2.0:
1. Navigate to `C:\Users\<your-username>\.wingetui`
2. Right-click the folder -> *Create* -> *Text document*
3. Rename the file to  `DisableAutoUpdateWingetUI`. Make sure to remove the `.txt` extension from the file name.
6. Uninstall WingetUI from the windows settings
7. Download and install WingetUI 2.2.0 from here: https://github.com/marticliment/WingetUI/releases/download/2.2.0/WingetUI.Installer.exe

## Explanation of the issue: 
This issue mostly happens due to a bug on .NET 8.0.3 (hence the downgrade to 8.0.2) or for the Windows App Sdk not being installed properly. I am working to fix this issue, but since it is not directly caused by WingetUI, it is hard to fix it.

**Comment by farzadoxo** [comment]
Great
